packages/rich-text/src/plugins/CommandPalette/components/CommandList.tsx

```diff
@@ -83,13 +92,13 @@ const CommandListItems = ({
 }) => {
   return (
     <>
-      {commandItems.map((command) => {
+      {commandItems.map((command, i) => {
         return 'group' in command ? (
-          <Group commandGroup={command} selectedItem={selectedItem} />
+          <Group key={i} commandGroup={command} selectedItem={selectedItem} />
```

Comment:
as every command should be unique, can't we use the command name instead of the index as key?

Reply:
Yes, you're right! Fixed